ci/Jenkinsfiles/build.groovy

```diff
@@ -70,6 +70,7 @@ pipeline {
     SCM_REF = "${getCommitSha1()}"
     VERSION = "${getVersion(REFERENCE_BRANCH)}"
     PERSISTENCE = "${BRANCH_NAME == REFERENCE_BRANCH}"
+    NUXEO_TEMPLATE_OVERRIDE = "${BRANCH_NAME == REFERENCE_BRANCH?'':'nuxeo.templates=default'}"
```

Comment:
Could add spaces before/after `?` and `:`.
Maybe add a comment to explain the override?